php/example_code/iam/tests/IamServiceTest.php

```diff
@@ -0,0 +1,296 @@
+<?php
+
+namespace Iam\Tests;
+
+use Exception;
+use Iam\IamService;
+use PHPUnit\Framework\TestCase;
+
+class IamServiceTest extends TestCase
+{
+    protected static $service;
+    protected static $uuid;
+    protected static $policy = "{
+                \"Version\": \"2012-10-17\",
+                \"Statement\": [{
+                    \"Effect\": \"Deny\",
+                    \"Action\": \"*\",
+                    \"Resource\": \"*\"
+            }]}";
+
+    public static function setUpBeforeClass(): void
+    {
+        self::$uuid = uniqid();
+        self::$service = new IamService();
+    }
+
+    public function testConstructor()
+    {
+//        self::$service = new IamService();
```

Comment:
Meant to uncomment?

Reply:
no, meant to delete since I changed how to set self::$service. But it still needs to be gone! fixed